org.eclipse.wildwebdeveloper/plugin.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.4"?>
 <plugin>
-   <extension-point id="org.eclipse.wildwebdeveloper.json.schema" name="Bind JSON Schema to filename pattern" schema="schema/json.schema.exsd"/>
```

Comment:
I see this got removed but there is a new .exsd file. Shouldn't it just be renamed?

Reply:
I did not add any new .exsd file, but as we need to keep the extension point, i will re-add the orginial .exsd file again.